Project-2/README.md

```diff
@@ -49,11 +118,11 @@ Note that we can drop the first universal quantifiers :)
 
 (c) ∀y(R(x, y) → ∃x(R(y, x) ∧ ∀y(R(x, y) → A(y)))) - For all y if there is some x thats Rs all instances of y then there is some x such that all instances of y Rs some x and for all y if x Rs y then y As
 
-Is this well-formed? I think not. For it to be well formed it should look closer to  ((∀y∃x(R(x, y) → R(y, x)) ∧ (R(x, y) → A(y)))
+Is this well-formed? I think not. For it to be well-fromed there should be another set of parenthesis to make it a conjunction additionally there is an unbound x
```

Comment:
Q4. Here we wanna use a formula to differ between two models, so you should save " ≥ 3R.T" (delete " P ⊓ " because P or D is not a concept in your interpretation) and delete "D ⊓ ≥ 2R.T". 

ρ = {(D, D2), (Perci, Perci2), (Moira, Moira2)} -- You should add (Theia, Moira 2), a bisimulation should map all elements of a model to those of another model. 

I find your right-side picture should be "Delaney 2" but not still "Delaney". 

"by defining the role t as ≥2 ∃t.⊤ in I1 ≥1 ∃t2.⊤ in I2" -- You should intead say, "I1 there is an element with 3 successors but in I2 there is an element with 2 elements"

Q5. "Role Assignments t={(D, Perci), (D,Moira), (D, Theia)}" This is wrong because in I1, you have only two successors according to your left-side graph.

" But we can distinguish them in ALCN by defining the role m as ≥2 ∃t.⊤ in I1 ≥1 ∃t2.⊤ in I2" -- No, you should say "cannot distinguish formula X in ALC and ALCN". You can also change the names to other names in I2, keeping the same result.

Reply:
Thank you, I think I fixed it...

Reply:
> Thank you, I think I fixed it...

Sorry I cannot see your revision. I just look through the newest file shown on your timeline of pull request

Reply:
Karl. My computer disconnect from the WiFi before I could commit my changes and after I posted this. It should be fixed now for real this time :)
Don't worry I only cried a little about losing my work

Reply:
@dmcnulty27 Oh shoot! I'm happy that you don't lose any effort you paid! :)